featuretools/utils/recommend_primitives.py

```diff
@@ -195,8 +196,9 @@ def _recommend_non_numeric_primitives(
                 for f_name in f.get_feature_names():
                     if len(matrix[f_name].unique()) > 1:
                         recommended_non_numeric_primitives.add(f.primitive.name)
-            except Exception:  # If error in calculating feature matrix pass on the recommendation
-                pass
+            except Exception as e:  # If error in calculating feature matrix pass on the recommendation
+                logger = logging.getLogger("featuretools")
+                logger.error(f"Exception in {f.primitive.name}: {str(e)}")
```

Comment:
What about a message like this?

"Exception with feature {f.get_name()} with primitive {f.primitive.name}: {str(e)}"

Reply:
Changed